Article/Article.js

```diff
@@ -3,24 +3,29 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = this.domElement.querySelector('.expandButton');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.innerText = 'expand'
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandButton.addEventListener('click', () => this.expandArticle(event));
   }
 
-  expandArticle() {
+  expandArticle(event) {
```

Comment:
Everything looks great but you could do an if statement here to change the text to close if the article is open and text to open if the article is closed.